src/test/kotlin/uk/gov/justice/digital/hmpps/personrecord/integration/MultiApplicationContextExtension.kt

```diff
@@ -0,0 +1,33 @@
+package uk.gov.justice.digital.hmpps.personrecord.integration
+
+import org.junit.jupiter.api.extension.BeforeAllCallback
+import org.junit.jupiter.api.extension.ExtensionContext
+import org.junit.jupiter.api.extension.ExtensionContext.Store.CloseableResource
+import org.springframework.boot.builder.SpringApplicationBuilder
+import uk.gov.justice.digital.hmpps.personrecord.HmppsPersonRecord
+
+class MultiApplicationContextExtension : BeforeAllCallback, CloseableResource {
+
+  internal val instance1: SpringApplicationBuilder = SpringApplicationBuilder(HmppsPersonRecord::class.java)
+    .profiles("test", "test-instance-1")
```

Comment:
The other ApplicationContext is initialised by IntegrationTestBase - not sure how well this works in practice....